messaging/src/main/java/org/axonframework/messaging/ResultAwareMessageDispatchInterceptor.java

```diff
@@ -0,0 +1,21 @@
+package org.axonframework.messaging;
+
+public interface ResultAwareMessageDispatchInterceptor<T extends Message<?>> {
```

Comment:
I think you've pushed this accidentally, if I recall some of our meetings last week correctly. ;-)